database/migrations/2023_05_30_164538_add_retry_attempts_and_retry_wait_time_field_to_scripts_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddRetryAttemptsAndRetryWaitTimeFieldToScriptsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('scripts', function (Blueprint $table) {
+            $table->unsignedSmallInteger('retry_attempts')->after('code')->default(60);
+            $table->unsignedSmallInteger('retry_wait_time')->after('code')->default(60);
```

Comment:
Defaults should be retry 0 and retry wait time 5, also change in script versions

Reply:
fixed